proposed/http-middleware/middleware-meta.md

```diff
@@ -222,8 +222,8 @@ and a delegate and must return a response. The middleware may:
 - Evolve the request before passing it to the delegate to execute the next
   available middleware.
 - Evolve the response received from the delegate before returning it.
-- Create and return a response without passing it to the delegate, thereby
-  preventing any further middleware from processing.
+- Create and return a response without further process delegation, thereby
+  preventing any inner middleware from processing.
```

Comment:
While `inner` refers to the onion image, it does not always fit to all frameworks. E.g. in [Strategility](https://zendframework.github.io/zend-stratigility/middleware/):

```php
// Hello-World middleware
$app->pipe('/', function ($req, $res, $next) {
    if (/* whatever */) {
        return $next($req, $res); // will dispatch FOO middleware
    }

    $res->getBody()->write('Hello world!');
    return $res;
});

// FOO middleware
$app->pipe('/', function ($req, $res, $next) {
    $res->getBody()->write('FOO!');
    return $res;
});
```

I believe the second middleware cannot be seen as an _inner_ middleware.